src/services/search.service.ts

```diff
@@ -10,21 +10,26 @@ import { CipherType } from '../enums/cipherType';
 import { FieldType } from '../enums/fieldType';
 import { UriMatchType } from '../enums/uriMatchType';
 import { SendView } from '../models/view/sendView';
+import { I18nService } from '../abstractions';
 
 export class SearchService implements SearchServiceAbstraction {
     private indexing = false;
     private index: lunr.Index = null;
+    private searchMinEdge = 2;
```

Comment:
Please consider variable name of, `searchableMinLength` instead of "searchMinEdge".